2022.11.0 (#24790)
home-assistant/home-assistant.github.io: source/_posts/2022-11-02-release-202211.markdown

- [New Integrations](#new-integrations)
- [Integrations now available to set up from the UI](#integrations-now-available-to-set-up-from-the-ui)
- [Need help? Join the community!](#need-help-join-the-community)
- [Breaking Changes](#breaking-changes)
- [All changes](#all-changes)

## Tile card

This release introduces a new Dashboard card: the "Tile"!

<img class="no-shadow" src='/images/blog/2022-11/tile-card.png' alt='Screenshot showing a couple of tile cards'>

The tile card is a lovely little card that provides a quick overview of an
entity. The icon can be tapped to toggle the entity, and the rest of the card
brings up the "more info" dialog.

[Read more about the Tile card in the documentation](/dashboards/tile).

This first iteration of the Tile card is one of the first cards added this
release, that brings in a new style of card into Home Assistant.

Curious to find out what else will be coming? Join us for the State
of the Open Home live stream. [Go to YouTube and click the bell 🔔](https://www.youtube.com/watch?v=D936T1Ze8-4)
so you get notified when we go live!

## Statistic card

More cards! We already have the [Statistics Graph card](/dashboards/statistics-graph),

> which can be used to display graph long-term statistics of an entity.

Reviewer: ```suggestion
which can be used to display long-term statistics of an entity.
```

> which can be used to display graph long-term statistics of an entity.

Reviewer 2: ```suggestion
which can be used to display graphs of long-term statistics of an entity.
```

[Check out the documentation](/docs/energy/water) for the hardware options
available to track your water usage or how to build your own water usage
sensors.

## Color temperatures in Kelvin

Another [WTH topic](https://community.home-assistant.io/t/wth-is-light-temperature-not-in-kelvin/467449)
that was voted for almost 150 times! Why doesn't Home Assistant use Kelvin
for color temperatures?

Well, a small piece of history: The Home Assistant story started using Philips
Hue a long, long time ago. And Hue uses mired for its color temperature,
not Kelvin.

But today, Kelvin is much more commonly used. It is even on packages of the
bulbs you buy. So, as of today, Home Assistant uses Kelvin too!

<img class="no-shadow" src='/images/blog/2022-11/kelvin.png' alt='Screenshot showing the UI now uses the Kelvin unit to set the color temperature'>

Kelvin is now the primary color temperature unit. This means all lights
and everything color temperature related shown in the user interface,
are now in Kelvin.

We still do support the good old mired. Mireds are still working and
automatically converted when needed, so this is not a breaking change 🎉

> Additionally, integration can now optionally use Kelvin directly if the

Reviewer 2: ```suggestion
Additionally, integrations can now optionally use Kelvin directly if the
```

## Long-term statistics in the entity dialogs

The "more info" dialogs of entities now use long-term statistics for
the graphs shown. It makes those dialogs a lot faster, especially on
mobile devices.

<img class="no-shadow" src='/images/blog/2022-11/long-term-stats-more-info-dialog.png' alt='Screenshot showing you can now set the day of the week from your user profile page'>

## The first day of the week

The start of the week can differ for many of us. Your week will start on
Saturday, Sunday, or Monday based on where you are, your locale, or maybe
personal preference. As pointed out in
[this WTH topic](https://community.home-assistant.io/t/wth-week-starts-on-sunday/468998).

Home Assistant will try to auto-detect the correct day to start the week
based on information provided by your browser. However, if it cannot detect it,
or in case you have a personal preference, you can now set it on your user profile.

<img class="no-shadow" src='/images/blog/2022-11/first-day-of-the-week.png' alt='Screenshot showing you can now set the day of the week from your user profile page'>

You can now set the day to start the week at {% my profile title="in your user profile" %}.

> Settings it means that all date inputs, date range pickers, schedules, calendars,

Reviewer 2: ```suggestion
Setting it means that all date inputs, date range pickers, schedules, calendars,
```

- Media player support was added to the [Jellyfin] integration. Thanks, [@Ongy]!
- The statistics card now supports "week" as a possible period. Thanks, [@mib1185]!
- The [Sun condition] can now handle setting both before & after at the same
  time. Thanks, [@janick]!
- The "forked-daapd" project was renamed to "Owntone". The integration was now
  renamed [Owntone] too! Thanks, [@uvjustin]!
- [Fully Kiosk Browser] now provides a service to remotely change URLs on your
  wall-mounted tablet. Thanks, [@cgarwood]!
- The [Xiaomi Miio] integration now supports the Xiaomi Purifier 4 Lite and
  the Roborock S7 MaxV. Thanks, [@alexdrl] & [@starkillerOG]!
- The [UniFi] integration now provides individual PoE control per port for
  the client and device PoE (switches are disabled by default).
  Thanks, [@Kane610]!
- When setting up a new [Generic Camera], you will be presented with a preview
  of the camera showing you if it works. Nice! Thanks, [@davet2001]!
- Support for `unique_id`s was added to the following (YAML capable) integrations:
  - [Bayesian], added by [@HarvsG]
  - [Min/Max], added by [@gjohansson-ST]
  - [Scrape], added by [@epenet]

> 

Reviewer 3: ```suggestion
 - The [HomeKit Controller] integration now supports encrypted Bluetooth notifications, reducing state change latency and the number of active BLE connections made to devices.

```

Author: Added 👍

a new `UPS Last Self Test` sensor reports time instead of the result.

You may need to replace `sensor.ups_last_self_test` with
`sensor.ups_self_test_result` in automations and scripts
referencing the `sensor.ups_last_self_test` entity.

([@definitio] - [#80773]) ([documentation](/integrations/apcupsd))

[@definitio]: https://github.com/definitio
[#80773]: https://github.com/home-assistant/core/pull/80773

{% enddetails %}

{% details "Brother Printer" %}

The `counter` and `remaining pages` attributes of the sensors
`drum remaining life`, `black drum remaining life`, `cyan drum remaining life`,
`magenta drum remaining life`, and `yellow drum remaining life` have been
migrated to their own dedicated sensor entities.

If you are using these attributes in your automation or script, you will need
to adopt those for this change.

([@bieniu] - [#79932]) ([documentation](/integrations/brother))

[@bieniu]: https://github.com/bieniu
[#79932]: https://github.com/home-assistant/core/pull/79932

{% enddetails %}

{% details "ClickSend TTS" %}

- The `caller` option from `ClickSend TTS` notify service has been removed, as
  it prevented calls from being actually made.

> - The default name to `clicksend_tts` instead of an empty name; otherwise

Reviewer 2: ```suggestion
- The default name has been changed to `clicksend_tts` instead of an empty name; otherwise
```

>   service appears as `notify.notify`. Now it will, by default, appear

Reviewer 2: ```suggestion
  the service would appear as `notify.notify`. Now it will, by default, appear
```

>   as `notify.clicksend_tts`. This might impact your automation or scripts.

Reviewer 2: ```suggestion
  as `notify.clicksend_tts`. This might impact your automation or script.
```

- The recipient's phone number now must match E.164 format.

([@CharlieBailly] - [#76669]) ([documentation](/integrations/clicksend_tts))

[@CharlieBailly]: https://github.com/CharlieBailly
[#76669]: https://github.com/home-assistant/core/pull/76669

{% enddetails %}

{% details "Coinbase" %}

The previously deprecated YAML configuration of the Open Exchange Rates
integration has been removed.

Open Exchange Rates is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely

> removed from your YAML configuration files.

Reviewer 2: ```suggestion
The previously deprecated YAML configuration of the Coinbase
integration has been removed.

Coinbase is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.
```

[@bdraco]: https://github.com/bdraco
[#79913]: https://github.com/home-assistant/core/pull/79913

{% enddetails %}

{% details "HomeKit Controller" %}

As older versions did not correctly set service identifier values, it may be
necessary to unpair and re-pair some BLE devices or to delete duplicate
entities manually.

If the service identifiers were loaded incorrectly when the device was paired
with an older version, operations might raise `StopIteration` or `RuntimeError`
errors. You can correct those by deleting the integration for that device,
and doing a one-time re-pairing.

([@bdraco] - [#80798]) ([documentation](/integrations/homekit_controller))

[@bdraco]: https://github.com/bdraco
[#80798]: https://github.com/home-assistant/core/pull/80798

{% enddetails %}

{% details "Hunter Douglas PowerView" %}

This change ceases creating battery sensors for hardwired shades. 
Existing sensor entities need to be manually deleted. 

([@kingy444] - [#81013]) ([documentation](/integrations/hunterdouglas_powerview))

[@kingy444]: https://github.com/kingy444
[#81013]: https://github.com/home-assistant/core/pull/81013

{% enddetails %}

{% details "IntelliFire" %}

>  Minor "breaking" change with very low impact. The backing unit and the API

Reviewer 2: ```suggestion
 The backing unit and the API
```

 of the device itself uses `0-4` as a flame height setting. Through discussion
 on the forums the community has decided `1-5` is a much more logical setting
 for flame height, and accordingly, this has been implemented.

 If you set the flame height using automations or scripts, you will  need to
 adapt those to this change.

([@jeeftor] - [#79901]) ([documentation](/integrations/intellifire))

[@jeeftor]: https://github.com/jeeftor
[#79901]: https://github.com/home-assistant/core/pull/79901

{% enddetails %}

{% details "Logger" %}

`logger` no longer changes the default log level of Home Assistant to `debug`
if used without a value for [`default`](/integrations/logger/#default). It
will now only change the default log level of Home Assistant from a
`warning` to something else if the `default` is explicitly specified.

If you do not specify `default` and want Home Assistant to have a

> default log level of `debug` will need to add this to your YAML configuration:

Reviewer 2: ```suggestion
default log level of `debug`, you will need to add this to your YAML configuration:
```
